### DISCUSSION TITLE
Option to avoid resolving domain to increase speed

### DISCUSSION THREAD

**Discussion by venom26** [projectdiscovery/nuclei#5379] [Ideas]
Hey team,
I hope you are doing well.

While we use nuclei with domains, there should be an option to ignore resolving each domain and directly start running templates on it instead.

This will help increase speed significantly.

# References:
https://www.moczadlo.com/2024/how-i-sent-500-million-http-requests-in-under-24h

Thank you

**Comment by tarunKoyalwar** [comment]
@venom26 , thanks for sharing but nuclei and other pd projects don't resolve domain everytime , it is only resolved one time and cached for later use, apart from that there are multiple checks and optimizations in place , if you are curious do take a look at [fastdialer](https://github.com/projectdiscovery/fastdialer)

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
